main.tf

```diff
@@ -395,33 +412,22 @@ data "external" "iac_tooling_version" {
   program = ["files/iac_tooling_version.sh"]
 }
 
-data "template_file" "sas_iac_buildinfo" {
-  template = file("${path.module}/files/sas-iac-buildinfo.yaml.tmpl")
-  vars = {
-    git-hash            = lookup(data.external.git_hash.result, "git-hash")
-    timestamp           = chomp(timestamp())
-    iac-tooling         = var.iac_tooling
-    terraform-version   = lookup(data.external.iac_tooling_version.result, "terraform_version")
-    provider-selections = lookup(data.external.iac_tooling_version.result, "provider_selections")
-    terraform-revision  = lookup(data.external.iac_tooling_version.result, "terraform_revision")
-    terraform-outdated  = lookup(data.external.iac_tooling_version.result, "terraform_outdated")
+resource "kubernetes_config_map" "sas_iac_buildinfo" {
```

Comment:
Again, not sure why we're changing patterns here? We use a template file and now a data path? Would like a discussion on this change.

Reply:
change needed to avoid permission issue with local file when running docker with local user.
Plus, it's a code simplification, imo

Reply:
Ok, we need to have a meeting to talk about this. Thanks.